### PR TITLE
Thunderbird update

### PR DESCRIPTION
**Pull request by piotrdrag**
This might be a one-time thing, or I might update Thunderbird regularly – I don't know yet.

### CODE REVIEW

File: mail/chrome/messenger/aboutSupportMail.dtd [hunk 1]
```diff
@@ -15,5 +15,6 @@
 <!ENTITY aboutSupport.accountsConnSecurity            "Bezpieczeństwo połączenia">
 <!ENTITY aboutSupport.accountsAuthMethod              "Metoda uwierzytelniania">
 <!ENTITY aboutSupport.accountsDefault                 "Domyślne?">
+<!ENTITY aboutSupport.identityName                    "Tożsamość">
```
Comment (splewako) [inline mail/chrome/messenger/aboutSupportMail.dtd:18]: Don't try to align by adding whitespaces in future please.

File: mail/chrome/messenger/messenger.properties [hunk 2]
```diff
@@ -315,6 +317,7 @@ editTagTitle = Edycja etykiety
 
 # For the virtual folder list dialog title %S is the name of the saved search folder
 editVirtualFolderPropertiesTitle = Edytuj właściwości folderu wyszukiwania %S
+virtualFolderSourcesChosen=#1 folder wybrany;Wybrano folderów: #1
```
Comment (splewako) [inline mail/chrome/messenger/messenger.properties]: This is wrong, see https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals

File: mail/chrome/messenger/prefs.properties [hunk 3]
```diff
@@ -54,3 +54,6 @@ identity-delete-confirm-button=Usuń
 choosefile = Wybierz plik
 
 forAccount = Zaawansowane ustawienia konta „%S”
+
+removeFromServerTitle=Potwierdzenie trwałego i jednocześnie automatycznego usuwania wiadomości
```
Comment (splewako) [inline mail/chrome/messenger/prefs.properties]: Why do you use "i jednocześnie" here?

File: mail/chrome/messenger/prefs.properties [hunk 4]
```diff
@@ -54,3 +54,6 @@ identity-delete-confirm-button=Usuń
...
+removeFromServer=To ustawienie będzie trwale usuwało stare wiadomości ze zdalnego serwera, jak i z lokalnej pamięci. Czy na pewno kontynuować?
```
Comment (splewako) [inline mail/chrome/messenger/prefs.properties]: I don't think that setting itself will remove anything.

File: mail/chrome/messenger/aboutDownloads.dtd [hunk 5]
```diff
@@ -13,5 +13,8 @@
 <!ENTITY cmd.open.accesskey "O">
 <!ENTITY cmd.removeFromHistory.label "Usuń z historii">
 <!ENTITY cmd.removeFromHistory.accesskey "U">
-<!ENTITY cmd.clearDownloads.label "Wyczyść pobrane">
-<!ENTITY cmd.clearDownloads.accesskey "e">
+<!ENTITY cmd.clearList.label "Wyczyść listę">
+<!ENTITY cmd.clearList.accesskey "c">
+<!ENTITY cmd.clearList.tooltip "Usuń wszystkie pobierania z listy, poza aktywnymi.">
+<!ENTITY cmd.searchDownloads.label "Szukaj…">
+<!ENTITY cmd.searchDownloads.key "S">
```
Comment (splewako) [inline mail/chrome/messenger/aboutDownloads.dtd]: We try not to change command keys.

File: mail/chrome/messenger/messenger.properties [hunk 6]
```diff
@@ -154,7 +155,8 @@ today       = Dzisiejsze
 yesterday   = Wczorajsze
 lastWeek    = Z ostatniego tygodnia
 twoWeeksAgo = Sprzed dwóch tygodni
-older=Starsze
+older       = Starsze
```
Comment (splewako) [inline mail/chrome/messenger/messenger.properties]: Do not touch unrelated lines.

File: mail/chrome/messenger/preferences/permissions.dtd [hunk 7]
```diff
@@ -23,5 +23,7 @@
 <!ENTITY allow.accesskey   "Z">
 <!ENTITY windowClose.key   "w">
 
-<!ENTITY button.close.label     "Zamknij">
-<!ENTITY button.close.accesskey "k">
+<!ENTITY button.cancel.label     "Anuluj">
+<!ENTITY button.cancel.accesskey "A">
```
Comment (splewako) [inline mail/chrome/messenger/preferences/permissions.dtd]: Doesn't it collide wtih address.accesskey?

File: mail/chrome/messenger/am-smime.properties [hunk 8]
```diff
@@ -11,7 +11,9 @@ ErrorCanNotSignMail=Cyfrowe podpisanie wiadomości było niemożliwe. Należy up
 # Strings used for in the prefs
 prefPanel-smime                 = Zabezpieczenia
 NoSigningCert                   = W Menedżerze certyfikatów nie odnaleziono żadnego ważnego certyfikatu, który mógłby zostać wykorzystany do cyfrowego podpisywania wysyłanych wiadomości.
+NoSigningCertForThisAddress     = W Menedżerze certyfikatów nie odnaleziono żadnego ważnego certyfikatu, który mógłby zostać wykorzystany do cyfrowego podpisywania wysyłanych wiadomości z adresem <%S>.
```
Comment (splewako) [inline mail/chrome/messenger/am-smime.properties]: z adresem „%S”

File: mail/chrome/messenger/am-smime.properties [hunk 9]
```diff
@@ -11,7 +11,9 @@ ErrorCanNotSignMail=Cyfrowe podpisanie wiadomości było niemożliwe. Należy up
...
 NoEncryptionCert                = W Menedżerze certyfikatów nie odnaleziono żadnego ważnego certyfikatu, który mógłby zostać wykorzystany przez inne osoby przy wysyłaniu szyfrowanych wiadomości do użytkownika tego konta.
+NoEncryptionCertForThisAddress  = W Menedżerze certyfikatów nie odnaleziono żadnego ważnego certyfikatu, który mógłby zostać wykorzystany przez inne osoby przy wysyłaniu szyfrowanych wiadomości do użytkownika tego konta na adres <%S>.
```
Comment (splewako) [inline mail/chrome/messenger/am-smime.properties]: z adresem „%S”

File: mail/chrome/messenger/aboutSupportMail.dtd (hunk 1 above)
Comment (piotrdrag) [inline mail/chrome/messenger/aboutSupportMail.dtd:18]: I tried to always keep the surrounding formatting. I can unify the spaces across the files, if you want (like I did for Fennec and got rid of the problem once and for all).

File: mail/chrome/messenger/messenger.properties (hunk 2 above)
Comment (piotrdrag) [inline mail/chrome/messenger/messenger.properties]: I copied it directly from suite.

Fixed here.

File: mail/chrome/messenger/prefs.properties (hunk 3 above)
Comment (piotrdrag) [inline mail/chrome/messenger/prefs.properties]: I copied it directly from suite.

"Potwierdzenie trwałego i automatycznego usuwania wiadomości" or "Potwierdzenie trwałego, automatycznego usuwania wiadomości"?

File: mail/chrome/messenger/aboutSupportMail.dtd (hunk 1 above)
Comment (splewako) [inline mail/chrome/messenger/aboutSupportMail.dtd:18]: Not introducing new ones is good enough for me but I don't have anything against separate patch removing them.

File: mail/chrome/messenger/prefs.properties (hunk 4 above)
Comment (piotrdrag) [inline mail/chrome/messenger/prefs.properties]: I copied it directly from suite.

Fixed.

File: mail/chrome/messenger/messenger.properties (hunk 2 above)
Comment (splewako) [inline mail/chrome/messenger/messenger.properties]: > > > virtualFolderSourcesChosen=#1 folder wybrany;Wybrano folderów: #1
> > 
> > This is wrong, see https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals
> 
> I copied it directly from suite.

@adrianer

File: mail/chrome/messenger/aboutDownloads.dtd (hunk 5 above)
Comment (piotrdrag) [inline mail/chrome/messenger/aboutDownloads.dtd]: Doesn't it add an underscore under a letter? If so, then "e" is not in "Wyczyść listę".

File: mail/chrome/messenger/prefs.properties (hunk 3 above)
Comment (splewako) [inline mail/chrome/messenger/prefs.properties]: I prefer "Potwierdzenie trwałego, automatycznego usuwania wiadomości" but both are better than existing one.

File: mail/chrome/messenger/messenger.properties (hunk 6 above)
Comment (piotrdrag) [inline mail/chrome/messenger/messenger.properties]: Again, tried to be consistent.

Fixed.

File: mail/chrome/messenger/prefs.properties (hunk 4 above)
Comment (splewako) [inline mail/chrome/messenger/prefs.properties]: > > > removeFromServer=To ustawienie będzie trwale usuwało stare wiadomości ze zdalnego serwera, jak i z lokalnej pamięci. Czy na pewno kontynuować?
> > 
> > I don't think that setting itself will remove anything.
> 
> I copied it directly from suite.

@adrianer

File: mail/chrome/messenger/preferences/permissions.dtd (hunk 7 above)
Comment (piotrdrag) [inline mail/chrome/messenger/preferences/permissions.dtd]: Indeed it does. Fixed.

File: mail/chrome/messenger/aboutDownloads.dtd (hunk 5 above)
Comment (splewako) [inline mail/chrome/messenger/aboutDownloads.dtd]: access key != command key

File: mail/chrome/messenger/am-smime.properties (hunk 8 above)
Comment (piotrdrag) [inline mail/chrome/messenger/am-smime.properties]: Fixed.

File: mail/chrome/messenger/am-smime.properties (hunk 9 above)
Comment (piotrdrag) [inline mail/chrome/messenger/am-smime.properties]: Fixed.

File: mail/chrome/messenger/aboutDownloads.dtd (hunk 5 above)
Comment (splewako) [inline mail/chrome/messenger/aboutDownloads.dtd]: In https://hg.mozilla.org/comm-central/diff/0977b5ddcf5e/mail/locales/en-US/chrome/messenger/aboutDownloads.dtd there is:

> +<!ENTITY cmd.searchDownloads.label                 "Search…">
> +<!ENTITY cmd.searchDownloads.key                   "F">

Comment (piotrdrag) [inline mail/chrome/messenger/aboutDownloads.dtd]: So which one are we talking about? cmd.clearList.accesskey, which has "accesskey" in the name, or cmd.searchDownloads.key, which is a new string?

Comment (splewako) [inline mail/chrome/messenger/aboutDownloads.dtd]: https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/key
https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/accesskey

File: mail/chrome/messenger/prefs.properties (hunk 3 above)
Comment (piotrdrag) [inline mail/chrome/messenger/prefs.properties]: Fixed.

File: mail/chrome/messenger/aboutDownloads.dtd (hunk 5 above)
Comment (piotrdrag) [inline mail/chrome/messenger/aboutDownloads.dtd]: Right. Fixed.

File: mail/chrome/messenger/aboutSupportMail.dtd (hunk 1 above)
Comment (piotrdrag) [inline mail/chrome/messenger/aboutSupportMail.dtd:18]: So I will do that in a new PR after we push this one. I guess we don't want to clutter this one with whitespace noise.

**Comment by splewako** [comment]
Thank you for the patches, landed as https://hg.mozilla.org/l10n-central/pl/pushloghtml?changeset=a34c15b56ced and https://github.com/aviarypl/mozilla-l10n-central-pl/compare/10a14d02a6a7...7876ef697dc8

File: mail/chrome/messenger/messenger.properties (hunk 2 above)
Comment (adrianer) [inline mail/chrome/messenger/messenger.properties]: Thx, fixed: https://hg.mozilla.org/l10n-central/pl/rev/5f2731da8299

File: mail/chrome/messenger/prefs.properties (hunk 4 above)
Comment (adrianer) [inline mail/chrome/messenger/prefs.properties]: Thx, fixed: https://hg.mozilla.org/l10n-central/pl/rev/5f2731da8299
